storage/interfaces.go

```diff
@@ -45,6 +46,7 @@ func DefaultSettings() *Settings {
 		URI:           "",
 		MigrationsURL: fmt.Sprintf("file://%s/postgres/migrations", basepath),
 		EncryptionKey: "",
+		SkipSSL:       false,
```

Comment:
I know that this is the default value, but would it make sense to add it to application.yml like we did with the api.skip_ssl_validation so that developers are aware they can change it?